README.md

```diff
@@ -81,3 +82,27 @@ plug "andreyorst/smarttab.kak" defer smarttab %{
     hook global WinSetOption filetype=(c|cpp) smarttab
 }
 ```
+
+## `autotab` configuration
+
+If you just want to set the behavior based upon your `editorconfig` settings, you can use the `autotab` setting:
+
+```kak
+hook global WinCreate .* %{
```

Comment:
This `WinCreate` hook confuses me a bit. Shouldn't it use `BufOpenFile` along with `BufNewFile` so editorconfig would apply to a certain filetypes, and then `autotab` will fetch the correct settings?

Reply:
You're right, partially. I changed it to `BufCreate` since this is the common hook run in both the `BufOpenFile` and `BufNewFile` cases; it's also what editorconfig uses for its hooks.